share/examples/bitcoin.conf

```diff
@@ -142,8 +142,11 @@
 # both prior transactions and several dozen future transactions.
 #keypool=100
 
+# Maintain coinstats index used by the gettxoutsetinfo RPC (default: 0).
+#coinstatsindex=1
+
 # Enable pruning to reduce storage requirements by deleting old blocks.
-# This mode is incompatible with -txindex and -rescan.
+# This mode is incompatible with -txindex, -coinstatsindex and -rescan.
```

Comment:
This is wrong. Indexes are compatible with pruning, if done right.

Reply:
If I understand correctly, @fjahr is working on that but for the moment it isn't the case yet

src/init.cpp#L888
```cpp
    argsman.AddArg("-prune=<n>", strprintf("Reduce storage requirements by enabling pruning (deleting) of old blocks. This allows the pruneblockchain RPC to be called to delete specific blocks, and enables automatic pruning of old blocks if a target size in MiB is provided.
            This mode is incompatible with -txindex, -coinstatsindex and -rescan. "
            "Warning: Reverting this setting requires re-downloading the entire blockchain. "
            "(default: 0 = disable pruning blocks, 1 = allow manual pruning via RPC, >=%u = automatically prune block files to stay under the specified target size in MiB)", MIN_DISK_SPACE_FOR_BLOCK_FILES / 
```


src/init.cpp#L888
```cpp
    // If using block pruning, then disallow txindex and coinstatsindex
    if (args.GetArg("-prune", 0)) {
        if (args.GetBoolArg("-txindex", DEFAULT_TXINDEX))
            return InitError(_("Prune mode is incompatible with -txindex."));
        if (args.GetBoolArg("-coinstatsindex", DEFAULT_COINSTATSINDEX))
            return InitError(_("Prune mode is incompatible with -coinstatsindex."));
    }
```

Reply:
#15946

Reply:
In principle it seems it should be doable: https://github.com/bitcoin/bitcoin/pull/19521/#discussion_r603607663 (not sure this GitHub link is working)